src/pages/test-report/components/driving-fault-summary/driving-fault-summary.ts

```diff
@@ -7,6 +7,7 @@ import { getTestData } from '../../../../modules/tests/test_data/test-data.reduc
 import { getDrivingFaultSummaryCount } from '../../../../modules/tests/test_data/test-data.selector';
 import { Subscription } from 'rxjs/Subscription';
 import { getTests } from '../../../../modules/tests/tests.reducer';
+import { NavController } from 'ionic-angular';
```

Comment:
We have a toolbar component now. So I think it would make more sense to put this functionality there.